docs.helm.sh/package.json

```diff
@@ -10,37 +10,37 @@
     "url": "git+https://github.com/helm/helm-www"
   },
   "devDependencies": {
-    "del": "^2.2.0",
+    "del": "^3.0.0",
     "font-awesome": "^4.7.0",
-    "foundation-sites": "^5.5.3",
+    "foundation-sites": "5.5.3",
```

Comment:
I had to make an exception for `foundation-sites`, as bumping this to the latest package version of the framework breaks the appearance, as v6 uses an entirely different set of HTML and CSS.

Reply:
I thought the caret operator was supposed to limit the package to the latest minor release, so `>= 5.5.3, < 6.0.0`. Is this not the case?

What about limiting this to the latest patch release via `~5.5.3`?